genfragments/ThirteenTeV/monoHiggs_UL/pythia8_hadronizer_nomatching_Haa_cff_CP5_PSweights_2018.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
```

Comment:
it should be the concurrent one from UL

Reply:
Can you please elaborate on this point? I do not fully understand what you meant.

Reply:
it's Pythia8ConcurrentHadronizerFilter